src/postman/operation.ts

```diff
@@ -1,16 +1,6 @@
-import {
-  HttpParamStyles,
-  HttpSecurityScheme,
-  IHttpHeaderParam,
-  IHttpOperation,
-  IHttpOperationRequestBody,
-  IHttpParam,
-  IHttpPathParam,
-  IHttpQueryParam,
-  IMediaTypeContent,
-} from '@stoplight/types';
+import { HttpSecurityScheme, IHttpOperation } from '@stoplight/types';
 import { IHttpOperationResponse } from '@stoplight/types/dist';
-import { Collection, CollectionDefinition, Item, RequestAuth, Url } from 'postman-collection';
+import { Collection, Item, RequestAuth, Url, CollectionDefinition } from 'postman-collection';
```

Comment:
You sure this is right? Normally imports are ordered alphabetically, why did linter move this to end?

Reply:
Almost sure I removed the import and then reimported when I understood it was a mistake.

Reply:
But linting should complain about unordered imports 🤔.